README.md

```diff
@@ -1,9 +1,69 @@
 # Polymer Linter
 
-Catch errors in your Polymer project before even running your code.
+Analyze your Polymer project for errors.
 
-## Work in progress
+## Usage
 
-Polymer Linter is being developed to replace [polylint][1], but is not yet complete.  For a working linter, use [polylint][1].
+The linter can be run in one of two ways: from the command line, or through our editor plugins.
+
+First though, it needs to be configured. Create a `polymer.json` file at the root of your project and ensure it has a "lint" field like so:
+
+```json
+{
+  "lint": {
+    "rules": ["polymer-2-hybrid"]
+  }
+}
+```
+
+You can specify either individual rules, or rule collections. See `polymer lint --help` for a full listing, or see [collections.ts](src/collections.ts) for details.
```

Comment:
Ideally if they've got an IDE plugin we could assist them there in choosing rules.

Removed the reference to the source here.